verticapy/tests_new/plotting/highcharts/machine_learning/test_highcharts_svm.py

```diff
@@ -0,0 +1,155 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+# Pytest
+import pytest
+
+# Standard Python Modules
+
+
+# Other Modules
+
+
+# Verticapy
+from verticapy.learn.svm import LinearSVC
+from verticapy.tests_new.plotting.conftest import get_xaxis_label, get_width, get_height
+
+# Testing variables
+COL_NAME_1 = "X"
+COL_NAME_2 = "Y"
+COL_NAME_3 = "Z"
+BY_COL = "Category"
+
+
+class TestHighchartsMachineLearningSVMClassifierPlot:
```

Comment:
The test classes for different ML algorithms are very similar. As I understand it, all test classes in this PR are for Highcharts only, and I suspect that you are planning to make similar ones for the other two plot libraries in your next PRs. Couldn't you make a base class to avoid so much code duplication?

Reply:
The reason I am doing like this is because I need the errors to be self-explanatory. When an error pops up, I can see the class and test function both. If there are different classes for each plotting library/tests, then it is easier to identify them.

Reply:
Have you tried it? There would be different subclasses for each plotting library/tests where they inherit one or several base classes. I think it would still write the name of the child class which will be executed as the test class.

Reply:
I just tried it. You are right. But then there is another issue. Not all the tests for a particular plot are identical. For example, i am testing the hovering label from plotly, but doing that from matplotlib is not possible as it does not exist. 
Also there are different values to test e.g. height and width. If i give 1 and 2 as height and width to plotly, it throws an error, whereas for matplotlib, that is acceptable as it takes input as inches. 
I will still try to refactor it. But that is looking like a lot of work with no significant benefit (other than reducing lines of code).

Reply:
A more concise code makes code maintenance easier. Imagine you may want to add an extra test for the libraries in the future, or there is an improvement you want to make; it would be sufficient to modify the base class instead of modifying so many different files.

Reply:
You can override the functions that need different implementations in different subclasses. Even fixtures can be overridden.

Reply:
As I briefly mentioned above, there are a lot of benefits in reducing the number of lines. It will take more intelligent work to write a concise code. In contrast, although writing plenty of duplicate codes may seem easy as copy-past, it would introduce a lot of headache. It would higher chance of mistakes, more difficult to make any improvement or fixing a problem in the future.